SparkConnector/README.md

```diff
@@ -2,83 +2,92 @@
 
 Helper to connect to CERN's Spark Clusters
 
-This extension is composed of a Python package named `sparkconnector`, which installs the nbextension and a NPM package named `@swan-cern/sparkconnector`
-for the JupyterLab extension.
+This extension is built as a Python module named `sparkconnector`, which simplifies the connection to Spark clusters.
+
+It installs:
+
+1. an nbclassic-extension
+1. a Jupyterlab extension
+1. an iPython extension
 
 
 ## Requirements
 
-* JupyterLab >= 3.0
+- JupyterLab >= 4.0.0
+- pyspark (not installed by default)
 
 ## Install
 
-Note: You will need NodeJS to install the extension.
+To install the extension, execute:
 
 ```bash
 pip install sparkconnector
-jupyter nbextension install sparkconnector --py
-jupyter nbextension enable  sparkconnector --py
-jupyter lab build
+jupyter nbclassic-extension install sparkconnector --py
```

Comment:
Is @PMax5 already using this new way of installing in his Alma9 image? Just making sure.

Reply:
Not yet. I still need to review that part as I hope many of those can be (and are already) automated. And we need to check if we need to disable some as well (i.e the spark connector for lab should not load automatically as well).